Block/Adminhtml/Button/AbstractButton.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Snowdog\Menu\Block\Adminhtml\Button;
+
+use Magento\Backend\Block\Widget\Context;
+
+class AbstractButton
+{
+    /** @var Context */
+    protected $context;
+
+    public function __construct(Context $context)
+    {
+        $this->context = $context;
+    }
+
+    protected function getUrl($route, $params = []): string
```

Comment:
i guess we can start using strong types (so we could start adding it in all new classes [params and method return types])?

Reply:
Sure, added strong types to params.